### ISSUE TITLE
UIBarButtons on iOS nearly invisible when using "open contact form" in dark mode.

### ISSUE THREAD

**Issue by ojhansson** [morenoh149/react-native-contacts#639]
It's almost impossible to see the "Cancel" and "Done" buttons as you can see in the image below.

![image](https://user-images.githubusercontent.com/31272853/132345863-a4de9837-c59f-4c3a-ae24-38cd11754bfc.png)

**Comment by elrony** [comment]
Another issue is that the navigation bar's tint colour is changed to be black, so if your app has a custom navigation bar the colour is replaced when the contact screen is presented and, when closed, it never goes back to its original colour.

I have tried removing the lines from 680-688 to let iOS deal with dark mode and it all works as expected.

Here for openContactForm:
https://github.com/morenoh149/react-native-contacts/blob/f74d3545405447fa61ad37c69a2418fc411a3b93/ios/RCTContacts/RCTContacts.m#L680

And here too for editExistingContact:
https://github.com/morenoh149/react-native-contacts/blob/f74d3545405447fa61ad37c69a2418fc411a3b93/ios/RCTContacts/RCTContacts.m#L863

From iOS 13 on all system "modals" automatically use dark/light mode. I have tested this on iOS 13/14/15 and it works correctly on all of them.

Should these lines be removed?
